src/lib.rs

```diff
@@ -412,6 +381,62 @@ impl Watch {
     }
 }
 
+struct WatchEventHandler {
+    watch: Watch,
+    command: Command,
+    child: Child,
+    command_start: Instant,
+}
+
+impl EventHandler for WatchEventHandler {
+    fn handle_event(&mut self, event: Result<Event, notify::Error>) {
+        match event {
+            Ok(event) => {
+                if self
+                    .watch
+                    .compare_event(&event, self.command_start.elapsed())
+                {
+                    log::trace!("Changes detected in {:?}", event);
+                    #[cfg(unix)]
+                    {
+                        let now = Instant::now();
```

Comment:
I would call it "killing_start" or something to make it obvious there is a bit of logic in here.

`now` is usually used when you want the put the exact same timestamp on different things that happens "now". For example, when updating a database with a lot of records, you usually want the update of all the records to have the same timestamp.